.coveragerc

```diff
@@ -1,7 +1,7 @@
 [report]
 omit =
     tests/*
-    settings/*
+    conf/*
```

Comment:
we are not including it...

Nevertheless, I need previously resolve the merge.